### PR TITLE
DOCS-5776-removed headers from strict validations

### PR DESCRIPTION
**Pull request by priyaayyar**
Removed references of validating headers as part of strict validations.

### CODE REVIEW

File: modules/ROOT/pages/api-proxy-landing-page.adoc [hunk 1]
```diff
@@ -33,6 +33,89 @@ API Proxy version 2.0.0 and later add native support for OAS API specification a
...
+* For Mule 4.x applications, both regular and strict validations are available.
+* Validations are performed for only WSDL and RAML or REST proxies.
+* Proxies are versioned in Mule 4.x applications. 
+* You can edit RAML or REST proxies from API Manager for RAML proxy version 1.2.0 and REST proxy version 2.1.0 and later.
+
+
+=== API Specifications for Strict Validations
+When enabling strict validations for Mule 4 applications, you must consider your API architecture. If your Mule application has a proxy or a load balancer in front of it, the web server might modify the client request to include additional headers. In such cases, the Mule application denies the incoming request due to a non-valid header error.
+
+When enabling strict validations for Mule 4 applications, you must consider your API architecture. If your Mule application has a proxy or a load balancer in front of it, the web server might modify the client request to include additional headers. In such cases, the Mule application denies the incoming request due to a non-valid header error.
```
Comment (sbehrends) [inline modules/ROOT/pages/api-proxy-landing-page.adoc]: This paragraph is duplicated

Comment (sbehrends) [reply modules/ROOT/pages/api-proxy-landing-page.adoc]: Actually this whole `API Specifications for Strict Validations` is referencing to strict headers validation. This section should be removed from `API Specifications for Strict Validations` until `Therefore, consider adding these headers in the API specification when using Edge.`

File: modules/ROOT/pages/api-proxy-landing-page.adoc [hunk 2]
```diff
@@ -33,6 +33,89 @@ API Proxy version 2.0.0 and later add native support for OAS API specification a
...
+With Mule 4, you can apply strict validations as well as regular validations. for your API proxies. Additionally, you can also edit proxies directly from API Manager based on the API proxy version. For more information, see the <<default-setting-mule-4,Default Setting by API Proxy Versions>> table in this section.
+
+=== Regular Validations
+
+Regular validations enable applications to compare a request with API specifications.  As part of the regular validations for REST or RAML proxies, the payload (schema), query parameters, URI parameters, headers, and form parameters are validated. For WSDL proxies, the envelope is validated against the structure provided in the WSDL file. If the proxy receives a parameter that is not specified in the definition, that parameter is irrespectively sent to the backend service.
+
+
+=== Strict Validations
+
+Strict validations enable applications to compare a request with proxy specifications and allow only those query parameters that are specified in the API definition. Currently, you can perform strict validations for only query parameters. This feature is available only for RAML and WSDL proxies in Mule 4.
```
Comment (sbehrends) [inline modules/ROOT/pages/api-proxy-landing-page.adoc]: We could add that the feature by default is disabled and can be enalbed only for RAML and WSDL proxies in Mule 4.

File: modules/ROOT/pages/api-proxy-landing-page.adoc [hunk 3]
```diff
@@ -33,6 +33,89 @@ API Proxy version 2.0.0 and later add native support for OAS API specification a
...
+  x-forwarded-for:
+    type: string
+  via:
+    type: string
+  x-forwarded-host:
+    type: string
+----
+
+Therefore, consider adding these headers in the API specification when using Edge.
+
```
Comment (sbehrends) [inline modules/ROOT/pages/api-proxy-landing-page.adoc:115]: Hi @priyaayyar, these last paragraphs are the ones we need to remove.

**Review by sbehrends — CHANGES_REQUESTED** [review]
Thanks for the PR @priyaayyar, please review the attached comments.

File: modules/ROOT/pages/api-proxy-landing-page.adoc (hunk 1 above)
Comment (priyaayyar) [reply modules/ROOT/pages/api-proxy-landing-page.adoc]: Done! Thanks!

File: modules/ROOT/pages/api-proxy-landing-page.adoc (hunk 2 above)
Comment (priyaayyar) [reply modules/ROOT/pages/api-proxy-landing-page.adoc]: Done! Thanks!

File: modules/ROOT/pages/api-proxy-landing-page.adoc (hunk 3 above)
Comment (priyaayyar) [reply modules/ROOT/pages/api-proxy-landing-page.adoc:115]: Ok Done!

**Review by dhirsch1 — APPROVED** [review]

**Review by sbehrends — APPROVED** [review]
LGTM, approved!

**Pull request merged**